installer/templates/phx_web/components/layouts.ex

```diff
@@ -101,17 +101,26 @@ defmodule <%= @web_namespace %>.Layouts do
   def theme_toggle(assigns) do
     ~H"""
     <div class="card relative flex flex-row items-center border-2 border-base-300 bg-base-300 rounded-full">
-      <div class="absolute w-[33%] h-full rounded-full border-1 border-base-200 bg-base-100 brightness-200 left-0 [[data-theme=light]_&]:left-[33%] [[data-theme=dark]_&]:left-[66%] transition-[left]" />
+      <div class="absolute w-[33.333%] h-full rounded-full border-1 border-base-200 bg-base-100 brightness-200 left-0 [[data-theme=light]_&]:left-[33.333%] [[data-theme=dark]_&]:left-[66.666%] transition-[left]" />
```

Comment:
I think tailwind 4 supports `w-1/3` and `left-1/3` now.

https://play.tailwindcss.com/5GD2qWzlTh

Reply:
nice!